dashboard/app/views/levels/editors/fields/_dsl_fields.html.haml

```diff
@@ -30,6 +30,11 @@
   .script_text
     = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
 
+  - if @level.is_a?(External)
+    = f.label "Associated Blocks (for embedded blocks)"
+    %p Picking a selection from the dropdown will NOT be sufficient to associate the block type with the level. Please add associated_blocks 'block type' to the DSL field.
```

Comment:
I don't know if I follow the instructions here. What _does_ picking a selection from this dropdown do?

Reply:
This is worded based on the [description for the video field](https://github.com/code-dot-org/code-dot-org/blob/bdb291227df311ff199614561ea0d8b78decdc72/dashboard/app/views/levels/editors/fields/_video.html.haml#L15) but I agree this could be clearer. I'll try to clean this up.